lib/src/models/frontegg_state.dart

```diff
@@ -15,9 +15,9 @@ class FronteggState {
     this.refreshToken,
     this.user,
     this.isAuthenticated = false,
```

Comment:
changing isLoading to false may create a race condition for customer on app launch.